counsel.el

```diff
@@ -1516,6 +1516,7 @@ TREE is the selected candidate."
 
 (add-to-list 'ivy-ffap-url-functions 'counsel-github-url-p)
 (add-to-list 'ivy-ffap-url-functions 'counsel-emacs-url-p)
+(add-to-list 'ivy-ffap-url-functions 'counsel-url-expansions)
```

Comment:
I think you want to use `counsel-url-expand` here?

Reply:
Ugh, yes. Must have been running stale code locally...